torch/_inductor/fx_passes/joint_graph.py

```diff
@@ -14,7 +15,7 @@
 def lazy_init():
     from .fuse_attention import _sfdp_init
 
-    with FakeTensorMode():
+    with detect_fake_mode() or FakeTensorMode():
```

Comment:
This is wrong as the patterns here are shared globally so you shouldn't arbitrarily pick the fake mode from a particular run; I actually have the more correct fix on a branch I am working on

Reply:
Okay - I'm going to close and re-assign to you then.

Reply:
@ezyang  could you share which PR? I want to follow when it lands as I have this PR checked out locally to unblock some quantization experiments.